docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM debian:stretch
+LABEL maintainer="Dash Evolution Developers <evodevs@dash.org>"
```

Comment:
I have been changing this to "Dash Developers <dev@dash.org>" in order to reflect our unified team/common goal, but not super critical.

Reply:
Updated label